onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -164,7 +170,9 @@ TEST_P(CApiTestWithProvider, simple) {
   std::vector<int64_t> expected_dims_y = {3, 2};
   std::vector<float> expected_values_y = {1.0f, 4.0f, 9.0f, 16.0f, 25.0f, 36.0f};
 
-  TestInference<PATH_TYPE, float>(*ort_env, MODEL_URI, inputs, "Y", expected_dims_y, expected_values_y, GetParam(), nullptr, nullptr);
+  void* library_handle = nullptr;
```

Comment:
>void* library_handle = nullptr [](start = 2, length = 30)

What happens next to this handle?

Reply:
This doesn't load the library. I added it so because we need to pass a parameter to the testing function

Reply:
Simplified everything - should be much cleaner now